mne/viz/_3d.py

```diff
@@ -636,6 +614,15 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
         Mayavi Scene in which to plot the alignment.
         If ``None``, creates a new 600x600 pixel figure with black background.
 
+        .. versionadded:: 0.18
```

Comment:
Version added should be below parameter definition

Reply:
whoops, my bad.